server/handler.go

```diff
@@ -109,7 +109,12 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 }
 
 func (h *Handler) handleCert(w http.ResponseWriter, req *http.Request) {
-	ac := h.CertManager.GetAugmentedChain()
+	digest := strings.TrimPrefix(req.URL.Path, h.CertPath+"/")
+	ac, err := h.CertManager.Cache.Read(digest)
+	if err != nil {
+		replyServerError(w, xerrors.Errorf("serializing cert-chain: %w", err))
```

Comment:
This error message doesn't look correct.

Reply:
Done